packages/ui/src/components/Tabs/STab.vue

```diff
@@ -0,0 +1,138 @@
+<script lang="ts">
+export default defineComponent({
+  name: 'STab',
+})
+</script>
+
+<script setup lang="ts">
+import { useTabsStateApi } from './api'
+
+const props = withDefaults(
+  defineProps<{
+    disabled?: boolean
+    borderRadius?: string
+  }>(),
+  {
+    disabled: false,
+    borderRadius: '',
+  },
+)
+
+const instance = getCurrentInstance()
+const { active, tabs, selectTab } = useTabsStateApi() || {
+  active: ref(0),
+  tabs: ref<any>([]),
+  selectTab: (t: number) => {},
+}

```

Comment:
I think `STab` component is not supposed to be used separately from `STabsPanel`, out of the context of its API. So, I think there is no reason to add a "default" value here - if injection is not possible, it should throw an error.

Reply:
Btw, you use `forceInject` in the `useTabsStateApi()`, so it will throw if there is an injection error anyway, and this default value will never be used.